.github/workflows/build_nugets.yml

```diff
@@ -35,6 +35,7 @@ jobs:
           -T ${{ matrix.toolset }}
           -DNATIVE_NUGET_PLATFORM_TAG="${{ matrix.arch_config.arch }}"
           -DBUILD_FLATBUFFERS=Off
+          -DBUILD_CSV=Off
```

Comment:
nit: `BUILD_CSV` is Off by default so can just omit in in these builds

Reply:
Fixed, thanks for reviewing!